server/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesRequestTests.java

```diff
@@ -93,4 +95,11 @@ public void testFieldCapsRequestSerialization() throws IOException {
             assertEquals(deserialized.hashCode(), request.hashCode());
         }
     }
+
+    public void testValidation() {
+        FieldCapabilitiesRequest request = new FieldCapabilitiesRequest()
+            .indices("index2");
+        ActionRequestValidationException exception = request.validate();
+        assertNotNull(exception);
```

Comment:
shall we also add a little test that validation fails when no indices are provided. I know you haven't changed the validation, but given that we are adding a test for it... :)

Reply:
It's actually okay to provide no indices -- if none are provided, then it defaults to all indices.

Reply:
right, I got confused, thanks for clarifying